.circleci/config.yml

```diff
@@ -670,6 +669,32 @@ workflows:
                 service-name: "${AWS_RESOURCE_NAME_PREFIX_CODEDEPLOY_FARGATE}-service"
                 cluster-name: "${AWS_RESOURCE_NAME_PREFIX_CODEDEPLOY_FARGATE}-cluster"
 
+      - aws-ecs/deploy-service-update:
+          name: codedeploy_fargate_test-update-and-wait-service-job
+          docker-image-for-job: circleci/python:3.4.9
+          requires:
+            - codedeploy_fargate_test-update-service-command
```

Comment:
I think this should be changed to `codedeploy_fargate_test-update-service-job`, so that this job will wait till the job named `codedeploy_fargate_test-update-service-job` has completed

Reply:
Thanks for the catch, made the update.